docs/en_us/platform_api/source/course_structure/index.rst

```diff
@@ -1,11 +1,25 @@
 .. _EdX Platform Course Structure API Version 0:
 
 #############################################
-Course Structure API Version 0
+Course Structure API Version 0 (Deprecated)
 #############################################
 
-.. toctree::
-    :maxdepth: 2
-    
-    overview
-    course_structure
+The course structure API version 0 is deprecated. EdX platform developers
+should not implement new client functions that use the course structure API
+version 0.
+
+You can get information about the courses offered by an edX platform
+installation by using the ``/api/courses/v1/courses/`` REST endpoint.
+
+You can get information about the parameters and return values of
+``/api/courses/v1/courses`` from the Django REST framework web page for that
+endpoint. For example, `https://courses.edx.org/api/courses/v1/courses/`_ provides information about the courses offered by edx.org.
+
+.. note::
+
+    The developer documentation available at `edX platform documentation`_ does
```

Comment:
maybe, The `edX platform documentation`_ does not currently include information... (just trying to reduce the number of times "documentation" is used in this note)